src/ProjectNeon/Assets/Features/Effects/WithCostOfBasicResource.cs

```diff
@@ -0,0 +1,17 @@
+﻿class WithCostOfBasicResource : Effect
+{
+    private Effect _effect;
+    private int _quantity;
+
+    public WithCostOfBasicResource (Effect effect, int quantity)
+    {
+        _effect = effect;
+        _quantity = quantity;
+    }
+
+    void Effect.Apply(Member source, Target target)
+    {
+        throw new System.NotImplementedException();
```

Comment:
This is not implemented.

Reply:
@SilasReinagel Sorry this class was a development prototype that I forgot do remove